client/src/modules/license/components/LicenseSelector.vue

```diff
@@ -0,0 +1,296 @@
+            sm:text-sm
+          ">
+        <option v-for="(commercial, cindex) in PodloveLicenseOptionCommercialList" :value="commercial.value"
+          :key="`commercial-${cindex}`">
+          {{ commercial.value }}
+        </option>
+      </select>
+    </div>
+    <div v-if="isJurisdicationNeeded" class="mb-3">
+      <label class="block text-sm font-medium text-gray-700">
+        License Jurisdiction
+      </label>
+      <select :value="getLicenseData().optionJurisdication?.name" @input="updateJurisdication($event)" class="
+            mt-1
+            block
+            w-full
+            py-2
+            px-3
+            border border-gray-300
+            bg-white
+            rounded-md
+            shadow-sm
+            focus:outline-none focus:ring-indigo-500 focus:border-indigo-500
+            sm:text-sm
+          ">
+        <option v-for="(jurisdication, jindex) in PodloveLicenseOptionJurisdication" :value="jurisdication.name"
+          :key="`jurisdiction-${jindex}`">
+          {{ jurisdication.name }}
+        </option>
+      </select>
+    </div>
+    <div class="mb-5">
+      <LicensePreview :license-data="getLicenseData()"></LicensePreview>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { PropType, defineComponent } from '@vue/runtime-core'
+import { injectStore, mapState } from 'redux-vuex'
+
+import { update as updateEpisode } from '@store/episode.store'
+import { update as updatePodcast } from '@store/podcast.store'
+import { selectors } from '@store'
+
+import LicensePreview from './LicenseView.vue'
+import PodloveButton from '@components/button/Button.vue'
+import Modal from '@components/modal/Modal.vue'
+
+import { PodloveLicense, PodloveLicenseVersion, PodloveLicenseOptionCommercial, 
+  PodloveLicenseOptionModification, PodloveLicenseOptionJurisdication, PodloveLicenseScope } from '../../../types/license.types'
+import { getLicenseFromUrl, getLicenseUrl } from '@lib/license'
+import episodeSagas from '@sagas/episode.sagas'
+
+const PodloveLicenseVersionList: {
+  key: string;
+  value: string;
+}[] = Object.entries(PodloveLicenseVersion).map(([key, value]) => ({ key, value }));
+
+const PodloveLicenseOptionCommercialList: {
+  key: string;
+  value: string;
+}[] = Object.entries(PodloveLicenseOptionCommercial).map(([key, value]) => ({ key, value }));
+
+const PodloveLicenseOptionModificationList: {
+  key: string;
+  value: string;
+}[] = Object.entries(PodloveLicenseOptionModification).map(([key, value]) => ({ key, value }));
+
+export default defineComponent({
+  components: { 
+    PodloveButton,
+    Modal,
+    LicensePreview,
+  },
+  data() {
+    return {
+      PodloveLicenseVersionList,
+      PodloveLicenseOptionCommercialList,
+      PodloveLicenseOptionModificationList,
+      PodloveLicenseOptionJurisdication
+    }
+  },
+  setup() {
+    return {
+      state: mapState({
+        episode_license_url: selectors.episode.license_url,
```

Comment:
In der Komponente verwende ich meistens CamelCase

Reply:
Done